<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QPushButton" name="Button_Off">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>310</y>
     <width>83</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Button_OFF</string>
   </property>
  </widget>
  <widget class="QSlider" name="servo_control_slide">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>430</y>
     <width>160</width>
     <height>18</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Orientation::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="Led_Off">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>83</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>LED_OFF</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBlue">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>250</y>
     <width>93</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Blue (D4)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Led_On">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>83</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>LED_ON</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Button_On">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>310</y>
     <width>83</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Button_ON</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="set_180">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>480</y>
     <width>93</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>180</string>
   </property>
  </widget>
  <widget class="QDial" name="servo_control_rotate">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>400</y>
     <width>50</width>
     <height>64</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>380</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Servo Control</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkYellow">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>210</y>
     <width>101</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Yellow (D8)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>0</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Flash Control</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>LED Control</string>
   </property>
  </widget>
  <widget class="QPushButton" name="flashButton">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>30</y>
     <width>83</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Flash</string>
   </property>
  </widget>
  <widget class="QPushButton" name="browseButton">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>30</y>
     <width>83</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Browse</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="set_90">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>480</y>
     <width>93</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>90</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>290</y>
     <width>311</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Buzzer Control through push button</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="set_0">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>480</y>
     <width>93</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="sketchPathEdit">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>30</y>
     <width>121</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Select Sketch File</string>
   </property>
  </widget>
  <widget class="QLabel" name="flashStatusLabel">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>60</y>
     <width>331</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkRed">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>170</y>
     <width>93</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Red (D13)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>130</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>LDR Control</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Light_sensor_On">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>170</y>
     <width>83</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>LDR_On</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Light_sensor_Off">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>170</y>
     <width>83</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>LDR_OFF</string>
   </property>
  </widget>
  <widget class="QPushButton" name="startDataButton">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>210</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Start Data Collection</string>
   </property>
  </widget>
  <widget class="QPushButton" name="stopDataButton">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>250</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Stop Data Collection</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="sensorValue">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>310</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="status_label">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>70</y>
     <width>381</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Status</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
